File: app/src/main/java/com/example/annada/eventfinder/model/BeenHere.java
package com.example.annada.eventfinder.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

public class BeenHere {
    @SerializedName("count")
    @Expose
    private Integer count;
    @SerializedName("marked")
    @Expose
    private Boolean marked;
    @SerializedName("lastCheckinExpiredAt")
    @Expose
    private Integer lastCheckinExpiredAt;

    public Integer getCount() {
        return count;
    }

    public void setCount(Integer count) {
        this.count = count;
    }

    public Boolean getMarked() {
        return marked;
    }

    public void setMarked(Boolean marked) {
        this.marked = marked;
    }

    public Integer getLastCheckinExpiredAt() {
        return lastCheckinExpiredAt;
    }

    public void setLastCheckinExpiredAt(Integer lastCheckinExpiredAt) {
        this.lastCheckinExpiredAt = lastCheckinExpiredAt;
    }
}

File: app/src/main/java/com/example/annada/eventfinder/model/BeenHere.kt
package com.example.annada.eventfinder.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

class BeenHere {
    @SerializedName("count")
    @Expose
    var count: Int? = null
    @SerializedName("marked")
    @Expose
    var marked: Boolean? = null
    @SerializedName("lastCheckinExpiredAt")
    @Expose
    var lastCheckinExpiredAt: Int? = null
}
